stdlib/private/SwiftPrivatePthreadExtras/SwiftPrivatePthreadExtras.swift

```diff
@@ -113,7 +113,7 @@ public class _stdlib_Barrier {
   var _pthreadBarrier: _stdlib_pthread_barrier_t
 
   var _pthreadBarrierPtr: UnsafeMutablePointer<_stdlib_pthread_barrier_t> {
-    return UnsafeMutablePointer(_getUnsafePointerToStoredProperties(self))
+    return _getUnsafePointerToStoredProperties(self).assumingMemoryBound(to: _stdlib_pthread_barrier_t.self)
```

Comment:
Oops. Must've thought I was in an overlay.
My emacs always does this:
return _getXXX()
.assumingMem(...)

but I think you want me to indent the second line which continues the statement.